src/main/java/io/cdap/e2e/pages/actions/CdfNameSpaceAdminActions.java

```diff
@@ -0,0 +1,119 @@
+  }
+
+  public static void openNamespaceAdminPage() {
+    ElementHelper.clickOnElement(CdfNameSpaceAdminLocators.namespaceAdminMenu);
+  }
+
+  public static void clickOnHamburgerMenu() {
+    ElementHelper.clickOnElement(CdfNameSpaceAdminLocators.hamburgerMenu);
+  }
+
+  public static void switchToNameSpace(String nameSpaceName) {
+    ElementHelper.clickOnElement(CdfNameSpaceAdminLocators.switchNameSpace(nameSpaceName));
+  }
+
+  public static void createProfileForNamespace(String nameSpaceName) {
+    ElementHelper.clickOnElement(CdfNameSpaceAdminLocators.createProfile(nameSpaceName));
+  }
+
+  public static void openNamespaceDropdown() {
+    ElementHelper.clickOnElement(CdfNameSpaceAdminLocators.namespaceDropdown);
+  }
+
+  public static void addNamespaceFromHamburgerMenu() {
+    ElementHelper.clickOnElement(CdfNameSpaceAdminLocators.addNamespace);
+  }
+
+  /**
+   * Enter KeyValue Pairs For Preference Property
+   *
+   * @param preferenceProperty @data-cy attribute value of preference Property. If
+   *                           preferenceProperty is present in
+   *                           {@link ConstantsUtil#DEFAULT_DATACY_ATTRIBUTES_FILE} then its data-cy
+   *                           is fetched from it else preferenceProperty is used as it is.
+   * @param keyValuePair       Actual json KeyValue Pairs string is fetched from
+   *                           {@link ConstantsUtil#DEFAULT_PLUGIN_PROPERTIES_FILE} with
+   *                           keyValuePair as key
+   */
+  public static void enterKeyValuePreferences(String preferenceProperty, String keyValuePair) {
+    String dataCyAttribute = PluginPropertyUtils.getPluginPropertyDataCyAttribute(
+        preferenceProperty);
+    if (dataCyAttribute == null) {
+      dataCyAttribute = preferenceProperty;
+    }
+    Map<String, String> properties =
+        JsonUtils.convertKeyValueJsonArrayToMap(PluginPropertyUtils.pluginProp(keyValuePair));
+    int index = 0;
+    for (Map.Entry<String, String> entry : properties.entrySet()) {
+      if (index != 0) {
+        ElementHelper.clickOnElement(CdfNameSpaceAdminLocators.locateAddRowButtonProperty(
+            dataCyAttribute, index - 1));
+      }
+      ElementHelper.sendKeys(CdfNameSpaceAdminLocators.locateKeyProperty(
+          dataCyAttribute, index), entry.getKey());
+      ElementHelper.sendKeys(CdfNameSpaceAdminLocators.locateValueProperty(
+          dataCyAttribute, index), entry.getValue());
+      index++;
+    }
+  }
+
+  public static void switchToNewNameSpace() {
+    ElementHelper.clickOnElement(CdfNameSpaceAdminLocators.switchToNameSpaceButton);
+  }
+
+  public static void verifySwitchToNewNamespace(String value) {
+    String expectedNameSpace = PluginPropertyUtils.pluginProp(value);
+    AssertionHelper.verifyElementContainsText(CdfNameSpaceAdminLocators.namespaceText,
+        expectedNameSpace);
+  }
+
+  public static void verifyElementIsDisplayed() {
```

Comment:
The method name seems to indicate that this method checks if any given element is displayed, where as it actually checks if the page header is displayed or not. Consider changing the method name to reflect its actual behavior.